docs/source/database/migrations.rst

```diff
@@ -126,6 +126,9 @@ you have to do the following:
 * Merge newly created stable branch migration to the master branch.
 * Also make the original master migration code idempotent (highly likely, it'll just be the same code as in stable
   branch).
+* An example for such a case is `src/middlewared/middlewared/alembic/versions/23.10/2023-11-10_23-19_add_exporting_table_for_reporting.py`
```

Comment:
Be more specific please. “For example, one can check if a table already exists by doing xxx, you can see file yyy as an example’.